src/Schema/Blueprint/ModifiesIndexes.php

```diff
@@ -29,6 +29,16 @@ public function sortKey($columns, $name = null)
         return $this->indexCommand('sortKey', $columns, $name);
     }
 
+    /**
+     * @param $columns
+     * @param $name
+     * @return \Illuminate\Support\Fluent
+     */
+    public function sortKeyDesc($columns, $name = null)
```

Comment:
since sortKey is a SingleStore specific thing (I am pretty sure Laravel doesn't support this normally) - what do you think about adding the direction as an optional parameter to the main `sortKey` method? Similar to the name argument?

Something like this:
```php
    public function sortKey($columns, $name = null, $direction = "asc")
```